src/CoreBundle/Service/MaintenanceModeService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+
+class MaintenanceModeService implements MaintenanceModeServiceInterface
```

Comment:
Likewise I'm not that happy about our massive usage of the Service suffix; this doesn't express anything and I'd like to avoid using it. How about something like MaintenanceModeActivator?

Reply:
Manager...

Reply:
After some discussion: "Service" fits the current interface contract best.